css/properties/appearance.json

```diff
@@ -121,7 +121,6 @@
             ],
             "support": {
               "chrome": {
-                "alternative_name": "base-select",
```

Comment:
If we remove the `alternative_name`, shouldn't we set `version_added: false`, given that `appearance: base` does not seem to be supported:

<img width="296" alt="image" src="https://github.com/user-attachments/assets/7c4be81d-86c6-4462-98a5-b7ee24b8d5e7" />